.circleci/config.yml

```diff
@@ -236,7 +239,38 @@ jobs:
                     path: build/
 
 workflows:
-    version: 2
     ci:
         jobs:
-            - build
+            -
+                build
+            -
+                docker/publish:
+                    deploy: false
+                    extra_build_args: '--build-arg GOPROXY=https://proxy.golang.org'
+                    image: banzaicloud/pipeline
+                    tag: $CIRCLE_BRANCH
+                    filters:
+                        branches:
+                            ignore: master
+            -
+                docker/publish:
+                    deploy: false
```

Comment:
Shouldn't this be true instead?

Reply:
Must be true, but it was just the first test build.